web/source/text/processor.ts

```diff
@@ -161,11 +161,11 @@ namespace com.keyman.text {
           if(special == EmulationKeystrokes.Backspace) {
             // A browser's default backspace may fail to delete both parts of an SMP character.
             this.keyboardInterface.defaultBackspace(Lkc.Ltarg);
-          } else if(special) {
+          } else if(special || DefaultOutput.isCommand(Lkc)) { // Filters out 'commands' like TAB.
```

Comment:
This fixes a minor issue I noted in testing on master, where hardware TABs weren't changing the focused element.